specification/ml/info/types.ts

```diff
@@ -32,9 +32,11 @@ export class NativeCode {
 }
 
 export class Limits {
```

Comment:
Not touched by you, but the memory limits should probably be `ByteSize` as well.